src/phoenix/experimental/evals/long_context_evaluators.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Optional
+
+from phoenix.experimental.evals import PromptTemplate
+from phoenix.experimental.evals.models import BaseEvalModel
+
+
+class Refiner:
+    def __init__(
+        self,
+        model: BaseEvalModel,
+        initial_prompt_template: PromptTemplate,
+        refine_prompt_template: PromptTemplate,
+        reduce_prompt_template: Optional[PromptTemplate] = None,
+    ) -> None:
+        self._model = model
+        self._initial_prompt_template = initial_prompt_template
+        self._refine_prompt_template = refine_prompt_template
+        self._reduce_prompt_template = reduce_prompt_template
+
+    def evaluate(self, chunks: List[str]) -> str:
+        if len(chunks) < 2:
+            raise ValueError("Must provide two or more documents")
```

Comment:
suggestion: Make the caller understand why this is important - e.g. "Refine strategy is not necessary when dealing with only 1 chunk as it is designed to ..... Consider using llm_classify instead." - Make the error message educational, informative, and actionable.

Reply:
Addressed in ca0b4a29